xesmf/tests/test_frontend.py

```diff
@@ -293,6 +303,15 @@ def test_regrid_dataarray_dask():
     # clean-up
     regridder.clean_weight_file()
 
+    # -----------------------------------------
+    # testing locstream in
+    regridder = xe.Regridder(ds_locs, ds_in, 'nearest_s2d', locstream_in=True)
+
+    outdata = regridder(ds_locs['lat']) 
+
+    # clean-up
+    regridder.clean_weight_file()
+
```

Comment:
(same as above) I would separate out of the tests for locstreams into different functions